app/templates/new_notification.html

```diff
@@ -0,0 +1,108 @@
+{% extends "./partials/base.html" %}
+{% block head %}
+  {{super()}}
+
+  <!-- CSS -->
+  <link rel="stylesheet" type="text/css" href="{{ url_for('static', path='/grid_style.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ url_for('static', path='/notification.css') }}" >
+
+{% endblock head %}
+{% block page_name %}Month View{% endblock page_name %}
+{% block body %}
+  <div class="pyldr-container">
+    {% include 'partials/calendar/navigation.html' %}
+    <div id="feature-settings">
+      {% include 'partials/calendar/feature_settings/example.html' %}
+    </div>
+    <div class="content">
+      {% block content %}
+        <header>
+            <div>
+                <div class="title">New Notifications</div>
+                <p class="s-paragraph">
+                    In this page you can view all of your new notifications.<br/>
+                    use the <button class="notification-btn btn-read"><ion-icon class="icon" name="eye-outline"></ion-icon></button>
+                    button to mark as read
+                    and the <button class="notification-btn btn-accept"><ion-icon class="icon" name="checkmark-outline"></ion-icon></button>
+                    and <button class="notification-btn btn-decline"><ion-icon class="icon" name="close-outline"></ion-icon></button>
+                    to accept and decline.
+                </p>
+            </div>
+            <div id="logo-div">
+                <a href="/calendar/month">
+                    <span class="title">PYLENDAR</span>
+                </a>
+            </div>
+        </header>
+        <div id="main">
+            <div><a href="{{ url_for('view_old_notifications') }}">Historical notifications</a></div>
+            {% if notifications %}
+            <div id="notifications-box">
+                <form action="{{ url_for('mark_all_as_read') }}" method="post">
+                    <input type="hidden" name="next" value="/notification">
+                    <button type="submit" class="btn btn-light btn-outline-primary btn-sm"
+                    {% if not new_messages %}disabled{% endif %} id="mark-all-ass-read">
+                        Mark all as read
+                    </button>
+                </form>
+                <div id="notifications">
+                    {% for n in notifications %}
+                        {% set type = n.__table__.name %}
+
+                        <div class="notification" id="{{ loop.index }}">
+                            {% if type == "invitations" %}
```

Comment:
What do you think about seperate `.jinja` files for in the contents inside the if-else blocks?